### ISSUE TITLE
Cannot locate file "for-each-folder--install-deps-and-compile.sh"

### ISSUE THREAD

**Issue by oldfartdeveloper** [JordanMartinez/purescript-jordans-reference#575]
### Section:

```
  Getting Started
    Install Guide
      #building-this-project
```

### Issue Type:

Bug (or my human error)

## Body

I cannot find the actual shell file because I don't know what I should have cloned in order to find it.  I'm not even sure I know what GitHub project or site this is supposed to come from.  Can you be more specific what I should have downloaded and where to look?

If this is my human error (i.e. I just didn't see it), then I worry that others might not be able to "see it either".

**Comment by JordanMartinez** [comment]
I'm pretty sure I deleted this file at one point. Essentially, it was doing almost the exact same thing that CI is now doing. I got tired of maintaining it, but I didn't update the above explanation.

- [spago--install.sh](https://github.com/JordanMartinez/purescript-jordans-reference/blob/latestRelease/.ci/spago--install.sh) installs all dependencies for each folder
- [spago--build.sh](https://github.com/JordanMartinez/purescript-jordans-reference/blob/latestRelease/.ci/spago--build-and-test.sh) build all folders
